acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/dsl/signer/TransactionSignerParamsSupplier.java

```diff
@@ -72,23 +72,23 @@ private File createKeyFile() {
       throw new RuntimeException(e);
     }
 
-    return createJsonFile("ethsigner_keyfile", data);
+    return createTmpFile("ethsigner_keyfile", data);
   }
 
   private File createVaultAuthFile() {
-    return createJsonFile("vault_authfile", HashicorpVaultDocker.vaultToken.getBytes(UTF_8));
+    return createTmpFile("vault_authfile", HashicorpVaultDocker.vaultToken.getBytes(UTF_8));
   }
 
-  private File createJsonFile(final String tempNamePrefix, final byte[] data) {
+  private File createTmpFile(final String tempNamePrefix, final byte[] data) {
     final Path file;
     try {
-      file = Files.createTempFile(tempNamePrefix, ".json");
+      file = Files.createTempFile(tempNamePrefix, ".file");
```

Comment:
why have a postfix at all? Assume someone passes in the whole filename?

Reply:
Good plan! Have to use .tmp then or null. Null also causes the suffix to be tmp ...
I'll go got null!

Reply:
I'll go with null!